Source/controls/devices/game_controller.h

```diff
@@ -8,6 +8,14 @@
 
 namespace devilution {
 
+enum GamepadLayout {

```

Comment:
Lets start new enums as scoped so that there is strict checking on them :)
```suggestion
enum class GamepadLayout {
```